indra_gpt/util/util.py

```diff
@@ -1,60 +1,128 @@
-from indra.statements import get_all_descendants, Statement
-
-
-def trim_stmt_json(stmt):
-    """Function to get rid of irrelevant parts of the indra statement json
-
-    Parameter
-    ---------
-    stmt : dict
-        The indra statement json object
-
-    Returns
-    -------
-    dict
-        The indra statement json object with irrelevant parts removed
-    """
-
-    stmt["evidence"] = [{"text": stmt["evidence"][0].get("text")}]
-
-    del stmt["id"]
-    del stmt["matches_hash"]
-
-    if 'supports' in stmt:
-        del stmt['supports']
-    if 'supported_by' in stmt:
-        del stmt['supported_by']
-
-    return stmt
-
-
-def post_process_extracted_json(gpt_stmt_json):
-    """Function to post process the extracted json from chatGPT
-
-    Parameters
-    ----------
-    gpt_stmt_json : dict
-        The extracted json from chatGPT
-
-    Returns
-    -------
-    dict
-        The post processed json or if there is a KeyError, the original json
+import json
+from typing import Union
+from indra_gpt.resources.constants import INPUT_DEFAULT
+import copy
+import random
+import logging
+logger = logging.getLogger(__name__)
```

Comment:
There should be two blank lines between the imports and the next pieces of code.
```suggestion


logger = logging.getLogger(__name__)
```